user-images/perftest/Dockerfile.with-cuda

```diff
@@ -0,0 +1,20 @@
+FROM nvidia/cuda:10.2-devel-ubuntu18.04
+
+ARG D_RDMA_CORE_TAG=v31.0
+ARG D_PERFTEST_TAG=4.4-0.29
+
+WORKDIR /root
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -yq install net-tools iproute2 apt-utils git autotools-dev autoconf libtool pciutils
+RUN DEBIAN_FRONTEND=noninteractive apt-get -yq install build-essential cmake gcc libudev-dev libnl-3-dev \
+    libnl-route-3-dev ninja-build pkg-config valgrind python3-dev cython3 python3-docutils pandoc
+
```

Comment:
same as above

Reply:
will do a dual stage. 
this is very similar to the non-cuda version i didnt want to make one docker file for both as it would needlessly complicate how these are being build (adding a base image and build for cuda args as docker does not support conditionals in Dockerfile per-se)